frontend/src/metabase/collections/containers/FormAuthorityLevelFieldContainer.tsx

```diff
@@ -34,10 +14,3 @@ function FormAuthorityLevelFieldContainer({
     <PLUGIN_COLLECTION_COMPONENTS.FormCollectionAuthorityLevelPicker name="authority_level" />
   );
 }
-
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default _.compose(
-  // Ensures there's data for the `collectionsMap` prop
```

Comment:
there is no `collectionMap` prop in the `FormCollectionAuthorityLevel` (or anywhere else in the codebase) anymore